plugins/keycloak-backend/src/providers/KeycloakOrgEntityProvider.ts

```diff
@@ -151,13 +151,13 @@ export class KeycloakOrgEntityProvider implements EntityProvider {
   }
 
   constructor(
-    private options: {
-      id: string;
-      provider: KeycloakProviderConfig;
-      logger: Logger;
-      userTransformer?: UserTransformer;
-      groupTransformer?: GroupTransformer;
-    },
+      private options: {
+        id: string;
+        provider: KeycloakProviderConfig;
+        logger: Logger;
+        userTransformer?: UserTransformer;
+        groupTransformer?: GroupTransformer;
```

Comment:
@riginoommen had a good point when discussing this PR offline. Instead of forming an opinion on instance-specific transformation requirements, we should provide a customization piece (additionally we can provide a sane default for the most common cases).

We already have these transformers here available, however they are never used (dead code). Instead of the transformation below, can you please propagate these two transformers to the `parseUser` and `parseGroup` functions in https://github.com/janus-idp/backstage-plugins/blob/main/plugins/keycloak-backend/src/lib/read.ts and let them transform the data?

Reply:
I think we can a customization piece to env example using regex

Reply:
I don't think we should force a regex transformation on user, instead offer an optional default and allow them to actually apply `userTransformer` and `groupTransformer`. Please redo this PR according to my previous comment.

Please do NOT modifying `entities` in `src/providers/KeycloakOrgEntityProvider.ts`. Instead export a default `userTransformer`/`userTransformer` AND propagate it to `src/lib/read.ts` so it gets applied.